cmd/consumerd/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/medibloc/panacea-dep-consumer/server"
+)
+
+func main() {
+	httpPtr := flag.String("listen-addr", "", "http server listen address")
+	grpcPtr := flag.String("grpc-addr", "", "grpc server listen address")
+	flag.Parse()
```

Comment:
I think you can check variable `listen-addr`.
If the value is empty, the server will throw an unexpected error.

Reply:
I applied in this #2. Sorry for confusing it.